package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "ship-components-upload",
+  "version": "0.1.0",
+  "description": "Upload button component",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "grunt server",
+    "build": "grunt build",
+    "test": "./node_modules/jest-cli/bin/jest.js src/js --coverage",
+    "coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ship-components/ship-components-upload.git"
+  },
+  "keywords": [
+    "react",
+    "button",
+    "upload"
+  ],
+  "author": "Sepand Assadi <sepand.assadi@sony.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ship-components/ship-components-upload/issues"
+  },
+  "homepage": "https://github.com/ship-components/ship-components-upload#readme",
+  "devDependencies": {
+    "attr-accept": "^1.0.3",
+    "autoprefixer": "^6.1.0",
+    "babel": "^6.0.15",
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.1.4",
+    "babel-eslint": "^4.1.5",
+    "babel-jest": "git+https://github.com/isuttell/babel-jest.git#master",
+    "babel-loader": "^6.1.0",
+    "babel-preset-es2015": "^6.1.4",
+    "babel-preset-react": "^6.1.4",
+    "classnames": "^2.2.0",
+    "codacy-coverage": "^1.1.3",
+    "coveralls": "^2.11.4",
+    "css-loader": "^0.22.0",
+    "eslint": "^1.9.0",
+    "eslint-loader": "^1.1.1",
+    "eslint-plugin-react": "^3.8.0",
+    "extract-text-webpack-plugin": "^0.9.1",
+    "grunt": "^0.4.5",
+    "grunt-contrib-clean": "^0.6.0",
+    "grunt-eslint": "^17.3.1",
+    "grunt-webpack": "^1.0.11",
+    "jest": "^0.1.40",
+    "jest-cli": "^0.7.1",
+    "jquery": "^3.1.0",
+    "lodash": "^4.15.0",
```

Comment:
Doesn't look like we use lodash so let's remove it. Maybe check to see if there are other packages we are not using and remove them as well